packages/lesswrong/lib/ultraFeed/ultraFeedPostHelpers.ts

```diff
@@ -0,0 +1,170 @@
+import { accessFilterMultiple } from "../utils/schemaUtils";
+import { performQueryFromViewParameters } from "@/server/resolvers/defaultResolvers";
```

Comment:
This file should not be in the `lib` directory if it's primarily for running server-side code like fetching stuff from the database (or from 3rd party integrations)